ucm2/USB-Audio/MOTU/M4-HiFi-Stereo-Mono.conf

```diff
@@ -0,0 +1,139 @@
+Include.pcm_split.File "/common/pcm/split.conf"
+
+Macro [
+	{
+		SplitPCM {
+			Name "m4_stereo_out"
+			Direction Playback
+			Channels 2
+			HWChannels 4
+			HWChannelPos0 FL
+			HWChannelPos1 FR
+			HWChannelPos2 FC
```

Comment:
I guess that it's not for the surround sound. I think that FL/FR should be used here, too.

Reply:
Changed to FL/FR